docs/writing-tests/accessibility-testing.md

```diff
@@ -73,7 +81,7 @@ Cycling through both stories, you will see that the `Inaccessible` story contain
 
 ### Configure
 
-Out of the box, Storybook's accessibility addon includes a set of accessibility rules that cover most issues. You can also fine tune the [addon configuration](https://github.com/storybookjs/storybook/tree/next/addons/a11y#parameters) or override [Axe's ruleset](https://github.com/storybookjs/storybook/tree/next/addons/a11y#handling-failing-rules) to best suit your needs.
+Out of the box, Storybook's accessibility addon includes a set of accessibility rules that cover most issues. You can also fine-tune the [addon configuration](https://github.com/storybookjs/storybook/tree/next/addons/a11y#parameters) or override [Axe's ruleset](https://github.com/storybookjs/storybook/tree/next/addons/a11y#handling-failing-rules) to best suit your needs.
```

Comment:
I think, because this is configured via `parameters`, it will _not_ work when using axe via the test-runner. Do we need to call that out? I could definitely see people expect it to "just work".

And, outside of this docs PR, is there anything we can do to bridge the gap? /cc @yannbf

Reply:
Yes. Discussed with @yannbf here https://github.com/storybookjs/test-runner/issues/97

Reply:
Thanks for the context @shilman! Appreciate it! I will follow up with both of you on this.